docs/contents/services.md

```diff
@@ -1,204 +1,223 @@
 # Services
 
 The following AWS service APIs have service controllers included in ACK or have
-controllers currently being built.
+controllers in one of our [several project stages][project-stages].
+
+[project-stages]: https://aws-controllers-k8s.github.io/community/releases#project-stages
+
+ACK controllers that have reached the `RELEASED` project stage will also be in
+one of our [maintenance phases][maint-phases].
+
+[maint-phases]: https://aws-controllers-k8s.github.io/community/releases#maintenance-phases
 
 For details, including a list of planned AWS service APIs, see the [Service
-Controller Release Roadmap](https://github.com/aws/aws-controllers-k8s/projects/1):
+Controller Release Roadmap](https://github.com/aws-controllers-k8s/community/projects/1):
 
 !!! note "IMPORTANT"
     There is no single release of the ACK project. The ACK project contains a
     series of service controllers, one for each AWS service API. Each
     individual ACK service controller is released separately. Please see the
-    documentation on [release criteria](releases.md) for information on how we
+    [release documentation][releases] for information on how we version and
     release ACK service controllers.
 
-| AWS Service | Current Status | Next Milestone
+[releases]: https://aws-controllers-k8s.github.io/community/releases
+
+| AWS Service | Project Stage | Maintenance Phase | Next Milestone
```

Comment:
For clarity:
Over time, the "Maintenance Phase" column can have multiple entries (one for each Major SemVer version series) for a service. And each of that entry can be in different Maintenance Phase.

Reply:
Yeah, good point. I'm going to need to rework this table (or, better yet, generate it automagically) once any controller has >1 major version series.